db/migrate/20241105175000_add_life_cycle_to_work_packages.rb

```diff
@@ -1,5 +1,8 @@
 class AddLifeCycleToWorkPackages < ActiveRecord::Migration[7.1]
   def change
-    add_reference :work_packages, :project_life_cycle, foreign_key: true, null: true
+    add_reference :work_packages,
+                  :life_cycle_step,
```

Comment:
I think `project_ life_cycle_step` should be more clear, though lengthier
If you change it `foreign_key` can be just `true`